bigquery_storage/to_dataframe/main_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import pytest
+
+
+@pytest.fixture
+def clients():
```

Comment:
Nit - it's a little weird that the third thing returned from something called "clients" is project_id. See note below; it may work out to not return project_id here and just pull it from the client.

Reply:
Done.